app/tests/spec/views/form.js

```diff
@@ -103,6 +103,14 @@ define(function (require, exports, module) {
         assert.isFalse(view.isErrorVisible());
       });
 
+      it('hides messeges when input value is changed', function () {
+        view.displayError('this is an error');
+        assert.isTrue(view.isErrorVisible());
+        view.$('#email').val('some@email.com');
+        view.enableSubmitIfValid();
+        assert.isFalse(view.isErrorVisible());
+      });
```

Comment:
We also need a test for `displaySuccess`. You can either reuse this same test that you added or create a new one below. 

Here's an existing test that might be helpful to you:

```js

      it('hides any previously displayed success messages', function () {
        view.displaySuccess('the success message');
        view.displayError('the error message');
        assert.isTrue(view.isErrorVisible());
        assert.isFalse(view.isSuccessVisible());
      });
```